extensions/notebook/src/dialog/managePackages/addNewPackageTab.ts

```diff
@@ -82,12 +82,12 @@ export class AddNewPackageTab {
 				}, {
 					component: this.newPackagesNameLoader,
 					title: localize('managePackages.packageNameTitle', "Package Name")
-				}, {
-					component: this.newPackagesVersionsLoader,
-					title: localize('managePackages.packageVersionTitle', "Package Version")
 				}, {
 					component: this.newPackagesSummaryLoader,
 					title: localize('managePackages.packageSummaryTitle', "Package Summary")
+				}, {
+					component: this.newPackagesVersionsLoader,
+					title: localize('managePackages.packageVersionTitle', "Supported Package Versions for Python {0}", this.jupyterInstallation.installedPythonVersion)
```

Comment:
do we need to account for conda as well, or is it generally still referred to as "Python"?

Reply:
Anaconda still uses Python underneath, and we definitely do version filtering for that too. I'll check if they call it "Anaconda Python" or something.

Reply:
Doesn't look like they make a distinction at the Python level, it's all about package management. We already have a blurb in the search bar about searching for Anaconda packages, so I'll just leave it likes this for now and we can change it later if people have problems with it.